library/ssl_misc.h

```diff
@@ -2656,14 +2656,14 @@ mbedtls_ssl_mode_t mbedtls_ssl_get_mode_from_ciphersuite(
     const mbedtls_ssl_ciphersuite_t *suite);
 #endif /* MBEDTLS_SSL_SOME_SUITES_USE_CBC_ETM */
 
-#if defined(PSA_WANT_ALG_ECDH)
+#if defined(PSA_WANT_ALG_ECDH) || defined(PSA_WANT_ALG_FFDH)
```

Comment:
Note: the function should be renamed as otherwise the guards look weird, but we already agreed that renaming could be done in a follow-up, so no change required here (just pointing out for reference).